sqlmesh/core/model/definition.py

```diff
@@ -2634,6 +2637,64 @@ def _validate_model_fields(klass: t.Type[_Model], provided_fields: t.Set[str], p
         raise_config_error(f"Invalid extra fields {extra_fields} in the model definition", path)
 
 
+def _validate_model_properties(meta_fields: dict[str, t.Any], path: Path) -> None:
+    # store for later validation of specific properties
+    model_properties: dict[str, t.Any] = {}
+
+    # validate that all properties kinds are key-value mappings
+    for kind in PROPERTIES:
```

Comment:
This is unnecessary. We already validate properties with this validator: https://github.com/TobikoData/sqlmesh/blob/main/sqlmesh/core/model/common.py#L425-L432